Microsoft.DotNet.Interactive/Events/ValueProduced.cs

```diff
@@ -11,17 +11,25 @@ public class ValueProduced : KernelEventBase
         public ValueProduced(object value,
             IKernelCommand command,
             bool isLastValue = false,
-            IReadOnlyCollection<FormattedValue> formattedValues = null) : base(command)
+            IReadOnlyCollection<FormattedValue> formattedValues = null,
+            string id= null,
+            bool isUpdatedValue = false) : base(command)
         {
             Value = value;
             IsLastValue = isLastValue;
             FormattedValues = formattedValues;
+            Id = id;
+            IsUpdatedValue = isUpdatedValue;
         }
 
         public object Value { get; }
 
         public bool IsLastValue { get; }
 
         public IReadOnlyCollection<FormattedValue> FormattedValues { get; }
+
+        public string Id { get; }
+
+        public bool IsUpdatedValue { get; }
```

Comment:
It seems like this should implicitly be true if an id is set. Would it not be a coding error if this is set to `true` while `id` is set to `null`?

Reply:
is false if id is not set, but can be false even if Id is set